resources/user-agents/browsers/uc-browser/uc-web-3.json

```diff
@@ -255,15 +177,6 @@
           ]
         },
         {
-          "match": "UCWEB\/*(#PLATFORM#SAMSUNG; GT-I8350*) U2\/* UCBrowser\/#MAJORVER#.#MINORVER#* U2\/*",
```

Comment:
This is the identical pattern (to the one above it). I triple checked the pattern, engines, device, platforms.  Tests still pass with it removed.